### DISCUSSION TITLE
TrueBeam-like to TOPAS parameters

### DISCUSSION THREAD

**Discussion by JoseRamosMendez** [OpenTOPAS/OpenTOPAS#77] [Announcements]
Hi everyone!
If you're a TrueBeam Linac user, you could find useful a new [python-interface](https://doi.org/10.1016/j.ejmp.2024.104485) to translate DICOM-RT information to TOPAS parameter files. It takes advantage of the time features to allow the simulation of 3D-CRT, IMRT step-and-shoot, and VMAT. Instructions and further details can be found in this [repository](https://github.com/OpenTOPAS/TPS2TOPAS_interface.git).
Cheers.

**Comment by rfhill** [comment]
This is really good news for those of us who are interested in using TOPAS in conventional radiotherapy. Going to try to get a student to test this out!

**Comment by WebbROMP** [comment]
Hi Jose,

I've created my own HDMLC for the TrueBeam, but would love to get a hold of this. it doesn't seem obviously available in the myvarian monte carlo section. Do i need to request it specifically?

**Comment by robin-physics** [comment]
Hello Jose

The paper is very well written and we are testing this out in our environment which is Varian Eclipse/Truebeam linacs.
In the paper, it mentions about the TOPAS extension for the geometry of the MLC.
It does seem that this is needed for some of the MLC options.
But there is no such file or data on the MyVarian website.

I could be wrong in understanding this but do you mean the propriety information on the MLC which is in larger document in the Monte Carlo section.

**Comment by JoseRamosMendez** [comment]
Hi there,
For getting access to the TrueBeam MLCs extension, please reach out Dr. Daren Sawkey at Varian.
![image](https://github.com/user-attachments/assets/6a4edf6e-3b7a-45bd-9041-35d6e6c2c01c)
Best.

**Reply by robin-physics** [reply]
hi Jose

I was wondering if you had any other contact details as we have not been able to get information back from Dr Daren Sawkey.

regards

Robin

**Comment by rfhill** [comment]
Thank you Jose very much for that.

Greatly appreciated.

 

Regards

 

Robin
